src/test/regress/Makefile

```diff
@@ -26,6 +26,8 @@ MULTI_REGRESS_OPTS = --inputdir=$(citus_abs_srcdir) $(pg_regress_locale_flags) -
 pg_upgrade_check = $(citus_abs_srcdir)/upgrade/pg_upgrade_test.py
 citus_upgrade_check = $(citus_abs_srcdir)/upgrade/citus_upgrade_test.py
 
+template_mx_files = $(shell find $(citus_abs_srcdir)/specs/ -name 'base*mx.spec')
+generated_mx_files = $(patsubst $(citus_abs_srcdir)/specs/base_isolation%,$(citus_abs_srcdir)/specs/isolation%,$(template_mx_files))
```

Comment:
I think it's nice to run this on all spec files and write them to `src/test/regress/build/specs/` that way there's a clear separation between source files and non source files, and no magic file prefixes (only magic directories).

Looking at the code of the isolation tester in the postgres repo you can pass an `--inputdir` flag, which should be changed from the default `.` to `./build`. We actually already set this flag for the vanilla tester. However, it seems that it will first try to find them in `{outputdir}/specs`. Since this would be our source specs directory I think you would have to rename that directory as well, easiest would be to rename to `spec` (without the s)

Reply:
In spec files, `#` means comment which is a directive in cpp. Currently we use `//` for comments in the base isolation files for mx(since cpp ignores `//`). I guess we can use `//` in other spec files too(it might be even better than having mixed comment symbol). But would that be hard for people to switch from `#` to `//` ?

Reply:
I think it's fine to switch comment symbol and indeed an advantage if it's the same symbol everywhere.